packages/components/nodes/multimodal/OpenAI/OpenAIVisionChain.ts

```diff
@@ -0,0 +1,288 @@
+import { ICommonObject, INode, INodeData, INodeOutputsValue, INodeParams } from '../../../src/Interface'
```

Comment:
rename the file name to `OpenAIMultiModalChain.ts`?

Reply:
Fixed in a new commit